frontend/tests/unit/components/inputs/BasicInput.spec.ts

```diff
@@ -1,20 +1,18 @@
 import { mount } from '@vue/test-utils';
 
-import Input from '@/components/inputs/Input.vue';
+import BasicInput from '@/components/inputs/BasicInput.vue';
```

Comment:
I'm also wondering if it would make sense to stub the vFocusOnMount directive here for the same reason as above. Not sure though as it would require additional work wherever we use the directive.

Reply:
In order to make this work, we will need to bump `@vue/test-utils` to `2.2.0` since stubbing directives (using `<script setup>`) is not supported in the versions below that.
https://github.com/vuejs/test-utils/releases/tag/v2.2.0

Working on it, lets see how much will break 😄

Reply:
✅